servicecontrol/servicecontrol-instances/remotes.md

```diff
@@ -116,6 +116,56 @@ class ServiceControl ServiceControlPrimary
 class ServiceControlAudit1,ServiceControlAudit2 ServiceControlRemote
 ```
 
+### Multi-transport deployments
+
+When using multiple transports the ServiceControl can be setup together with the [MessagingBridge](/nservicebus/bridge/) to allow the entire system to be managed via a single ServicePulse/ServiceInsight installation.
+
+```mermaid
+graph TD
+primaryA -. connected to .-> auditA
+primaryA -. connected to .-> auditB
+primaryA -. connected to .-> auditC
+
+subgraph Transport A
+ServicePulse[ServicePulse] -. connected to .-> primaryA
+ServiceInsight[ServiceInsight] -. connected to .-> primaryA
+endpointsA[endpoints] -- send errors to --> errorQueueA[error queue]
+endpointsA -- send audits to --> auditQueueA[audit queue]
+errorQueueA -- ingested by --> primaryA[ServiceControl<br/>primary]
+auditQueueA -- ingested by --> auditA[ServiceControl Audit<br/>remote]
+end
+
+subgraph Transport B
+endpointsB[endpoints] -- send errors to --> errorQueueB[error queue]
+endpointsB -- send audits to --> auditQueueB[audit queue]
+errorQueueB -- ingested by --> bridgeB[Bridge]
+auditQueueB -- ingested by --> auditB[ServiceControl Audit<br/>remote]
+bridgeB -- forwards --> errorQueueA
+end
+
+subgraph Transport C
+endpointsC[endpoints] -- send errors to --> errorQueueC[error queue]
+endpointsC -- send audits to --> auditQueueC[audit queue]
+errorQueueC -- ingested by --> bridgeC[Bridge]
+auditQueueC -- ingested by --> auditC[ServiceControl Audit<br/>remote]
+bridgeC -- forwards --> errorQueueA
+end
+
+classDef Endpoints fill:#00A3C4,stroke:#00729C,color:#FFFFFF
+classDef Bridge fill:#a8a032,stroke:#00729C,color:#FFFFFF
+classDef ServiceInsight fill:#878CAA,stroke:#585D80,color:#FFFFFF
+classDef ServicePulse fill:#409393,stroke:#205B5D,color:#FFFFFF
+classDef ServiceControlPrimary fill:#A84198,stroke:#92117E,color:#FFFFFF,stroke-width:4px
+classDef ServiceControlRemote fill:#A84198,stroke:#92117E,color:#FFFFFF
+
+class endpointsA,endpointsB,endpointsC Endpoints
+class bridgeB,bridgeC Bridge
+class ServiceInsight ServiceInsight
+class ServicePulse ServicePulse
+class primaryA,primaryB ServiceControlPrimary
+class auditA,auditB,auditC ServiceControlRemote
+```
```

Comment:
For reference, this renders as:

![image](https://github.com/Particular/docs.particular.net/assets/677704/a3bc0d2d-602d-4900-90e6-bdb9bed09358)

For my own understanding at least, when I retry a message using ServicePulse, how does this work? Wouldn't the bridge have to be somehow used to shuffle the message back to relevant transport and endpoint?

Reply:
the bridge creates proxy queues in all transports so the retry goes to a local queue in transport A that the bridge then picks up and moves to the corresponding queue in the transport its bridging. Does that make sense?

Reply:
I think that it would clutter the diagram to much to get that in there?

Reply:
I get it now, thanks! I had a think about how to represent all of that in this diagram neatly but couldn't come up with anything and, in it's current form, it calls out the extra work required (on top of putting the bridge in place for the transports) to use a single ServicePulse, so I guess it makes sense as-is.